dev-docs/source/ReleaseChecklist.rst

```diff
@@ -148,6 +159,11 @@ Tuesday, 4 days
 
 *  Review the complete set of release notes to make sure there are no glaring mistakes.
 
+After Release
+-------------
+
+* Remove all separate release note files and sub-file structure to leave just the upper level release notes e.g. diffraction.rst, index.rst, framework.rst etc.
```

Comment:
Would it be possible to do this as one of the final steps in preparing to tag the release instead of after? It leaves the source for the release in a clean state then.